### PR TITLE
Feature/Three-slice view

### PR DESCRIPTION
**Pull request by tdlinh2712**
Team 23:
This PR allows a three-slice view display (axial, coronal, and sagittal). To use this feature, simply click on the four-square icon in the toolbar. To switch back to one-window view, click on the square icon. 

Note that this PR does not contain the cutline and ROI display for sagittal and coronal views in 3-slice view.

### CODE REVIEW

**Review by Tamookk — APPROVED** [review]
Found no problems during testing. Impressive work.
- Team 14

File: src/constants.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+DEFAULT_WINDOW_SIZE = 512
+INITIAL_ONE_VIEW_ZOOM = 1
+INITIAL_FOUR_VIEW_ZOOM = 0.5
```
Comment (DuyTungHa) [inline src/constants.py:3]: PEP-8 end of line missing

**Review by DuyTungHa — APPROVED** [review]
Please fix the PEP-8 issue in contants.py

**Review by JThomas2020 — APPROVED** [review]
Team 19

No bugs found, really good stuff

**Pull request merged**